nni/experiment/config/convert.py

```diff
@@ -105,7 +105,7 @@ def _handle_training_service(ts, data):
                 'gpuIndices': _convert_gpu_indices(machine.get('gpuIndices')),
                 'maxTrialNumPerGpu': machine.get('maxTrialNumPerGpu'),
                 'useActiveGpu': machine.get('useActiveGpu'),
-                'preCommand': machine.get('trialPrepareCommand')
+                'pythonPath': machine.get('trialPrepareCommand')
```

Comment:
`trialPrepareCommand` -> `trialPythonPath`
`trial_prepare_command` -> `trial_python_path`

also in other files, like `experiment_config.rst`.

Reply:
Done.